odoo/cli/help.py

```diff
@@ -0,0 +1,25 @@
+from .command import Command, commands, find_command, load_addons_commands, load_internal_commands
+from pathlib import Path
+import sys
+
+
+class Help(Command):
+    """ Display the list of available commands """
+
+    def run(self, args):
+        load_internal_commands()
+        load_addons_commands()
+        padding = max(len(cmd) for cmd in commands) + 2
+        command_list = "\n    ".join([
+            "    {}{}".format(name.ljust(padding), (command.__doc__ or "").strip())
+            for name in sorted(commands)
+            if (command := find_command(name))
+        ])
+        print(  # noqa: T201
+            "Odoo CLI, use '{odoo_bin} --help' for regular server options.\n"
+            "\n"
+            "Available commands:\n"
+            f"    {command_list}\n"
+            "\n"
+            f"Use '{Path(sys.argv[0]).name} <command> --help' for individual command help.\n"
```

Comment:
For readability, can you assign `prog_name = Path(...)` can just use it in this text?